src/registerToolbar.ts

```diff
@@ -142,7 +142,7 @@ export default function registerToolbar() {
     ToolbarItemFactory.register({
         name: ToolbarItemName.ExportProject,
         tooltip: strings.editorPage.toolbar.exportProject,
-        icon: "fa-external-link-square-alt",
+        icon: "fas fa-file-export",
```

Comment:
This should be the same as it originally had been until your other feature gets merged in. We'd like to keep the features separate, so this should match with the current master for now.